### PR TITLE
[guilib] The output buffer of IImage::Decode should be a const pointe…

### PR DESCRIPTION
**Pull request by ace20022**
…r rather than a pointer to a const value.

Found because of @Paxxi ;)

### CODE REVIEW

**Comment by ace20022** [comment]
@Paxxi ping

**Comment by MrMC** [comment]
missed a few in xbmc/cores/omxplayer/OMXImage.cpp and xbmc/cores/omxplayer/OMXImage.h

**Comment by ace20022** [comment]
Thx, but that's not an iimages.

jenkins build this please

File: xbmc/guilib/JpegIO.h [hunk 1]
```diff
@@ -42,7 +42,7 @@ class CJpegIO : public IImage
   static bool           CreateThumbnailFromSurface(unsigned char* buffer, unsigned int width, unsigned int height, unsigned int format, unsigned int pitch, const std::string& destFile);
   void           Close();
   // methods for the imagefactory
-  virtual bool   Decode(const unsigned char *pixels, unsigned int pitch, unsigned int format);
+  virtual bool Decode(unsigned char* const pixels, unsigned int pitch, unsigned int format);
```
Comment (da-anda) [inline xbmc/guilib/JpegIO.h]: indention seems wrong - at least on github webview

**Comment by Paxxi** [comment]
Looks good, go ahead and merge

**Pull request merged**